src/utils.js

```diff
@@ -239,3 +239,17 @@ export const isValidDate = (date) => {
 
   return Boolean(formattedValue.length <= 10);
 };
+
+export const useScrollToHashElement = () => {
```

Comment:
i think it's better to move to `hooks.js` only that inner `useEffect` without export either move export to global `hooks.js`

Reply:
Done